### ISSUE TITLE
A11y_Manage.IIS.Net_Installation Popup_Screen Reader: Heading levels are not programatically identifiable

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/IIS.WebManager#211]
**Environment:**
https://manage.iis.net/connect
Browser: Microsoft Edge 42.17134.1.0, Google Chrome 69.0.3497.100

**MAS ID Violated:**
MAS 1.3.1, MAS 2.4.6 

**Repro Steps:**
1. Hit the URL "https://manage.iis.net" 
2. Tab till "Download Microsoft IIS Administration" button and press enter and run "IISAdministrationSetup.exe" file
3. Turn on narrator in scan mode
4. Verify narrator is reading the heading levels or not in the agreement

**Actual Result:**
Narrator is not reading the heading levels in the agreement

**Expected Result:**
Narrator should read the heading levels in the agreement

**User Impact:**
Users with disabilities will finds difficult while installing if narrator does not specify the heading levels

**Note:**
Headings are not programmatically identifiable in the entire agreement

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0cf80872-800d-4643-90cf-0c1c3d3e6260}

**Attachment:**
[Bug_211 _headings are not programatically identifiable.pptx](https://github.com/Microsoft/IIS.WebManager/files/2433895/Bug_211._headings.are.not.programatically.identifiable.pptx)

**Comment by jhkimnew** [comment]
Low priority because this page won't be used in WAC mode.

**Comment by jhkimnew** [comment]
Closing...
This is a setup UI, which is not exposed to users anymore.

**Comment by Tanushagottipati** [comment]
GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #ScreenReader; #MAS1.3.1; #C+E-H; #.NETWebsites; #Chrome; #NVDA; MAS2.4.6; #Headings

**Comment by prabhudevu** [comment]
**GitHubTags:**#A11ySev3;
